### PR TITLE
Hw unique key conf

### PR DESCRIPTION
**Pull request by joerchan**

### CODE REVIEW

File: include/hw_unique_key.h [hunk 1]
```diff
@@ -22,34 +22,20 @@ extern "C" {
...
-#ifdef NRF5340_XXAA_APPLICATION
-	#define HUK_HAS_CC312
-#endif
-#endif /* __ZEPHYR__ */
+#define HUK_HAS_KMU   CONFIG_HW_HAS_NRF_KMU
+#define HUK_HAS_CC310 CONFIG_HW_HAS_NRF_CC310
+#define HUK_HAS_CC312 CONFIG_HW_HAS_NRF_CC312
 
-#ifdef HUK_HAS_CC310
+#if defined(HUK_HAS_CC310)
```
Comment (oyvindronningstad) [inline include/hw_unique_key.h:40]: I'm not sure this will work now. Won't it always be defined?

**Review by oyvindronningstad — CHANGES_REQUESTED** [review]

File: include/hw_unique_key.h (hunk 1 above)
Comment (joerchan) [reply include/hw_unique_key.h:40]: You're right. Will fix.

File: include/hw_unique_key.h [hunk 2]
```diff
@@ -22,34 +22,26 @@ extern "C" {
 
 #include <stdbool.h>
 
-#ifdef __ZEPHYR__
+#if defined(__NRF_TFM__)
```
Comment (oyvindronningstad) [inline include/hw_unique_key.h:25]: Should this be 
```suggestion
#if !defined(__NRF_TFM__)
```

Comment (joerchan) [reply include/hw_unique_key.h:25]: No, I want to include the header when building in TF-M.

**Pull request merged**